src/layouts/SiteLaunchPad/SiteLaunchPad.tsx

```diff
@@ -241,15 +261,22 @@ export const SiteLaunchPadPage = (): JSX.Element => {
 
   return (
     <>
-      {isSiteLaunchEnabled(siteName, role) ? (
-        <SiteLaunchPad
-          pageNumber={pageNumber}
-          setPageNumber={setPageNumber}
-          handleDecrementStepNumber={handleDecrementStepNumber}
-          handleIncrementStepNumber={handleIncrementStepNumber}
-        />
+      {isLoaded ? (
```

Comment:
Request for ways to do this better. Ideally this should be a simple wrap over
```
<Skeleton isLoaded> 
   {isSiteLaunchEnabled(siteName, role) ? <SiteLaunchPad/>: <Redirect/>}
</Skeleton> 
```
But somehow this immediately redirects even when it is currently loading, leading to this is imperative code here.

Reply:
The above Skeleton code looks correct. Have you looked at React dev tools when the loading happens, it is changing at the right point?

Reply:
So when logging here, notice that there is at least 1 frame in which `hello` is printed out even when the value of `isLoaded` is false. Not too sure the exact reason, but this is suggestive of the `Skeleton` component not working as intended. 

![Screenshot 2023-08-16 at 10 26 43 AM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/7831512d-d111-4501-93b1-719d5ad78ed5)

<img width="349" alt="Screenshot 2023-08-16 at 10 26 50 AM" src="https://github.com/isomerpages/isomercms-frontend/assets/42832651/20c47778-07d6-46b1-90b7-3d23fb7d76d9">

Reply:
You need to do `isLoaded && <Redirect to={`/sites/${siteName}/dashboard`} />`
else it will evaluate the Redirect before isLoaded turns to true

Reply:
ok done 79cc921c94c8ee1c2f0c4f23e7ca4b624beedeee